nss-cohort-36/keahua-arboretum-net-surfers
# Sp finish print report

# Description

[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-finish-print-report`
4. make all kinds of combinations of animals and biomes and try printing display reports of them

# Checklist:
